sevsnpmeasure/id_block.py

```diff
@@ -157,8 +159,8 @@ def marshal_ec_public_key(priv_key: ec.EllipticCurvePrivateKey) -> bytes:
     qy = y.to_bytes(0x48, byteorder="little")
     result = ECPublicKey(
         curve=CurveP384,
-        qx=qx,
-        qy=qy
+        qx = Qx.from_buffer_copy(qx),
+        qy = Qy.from_buffer_copy(qy),
```

Comment:
Apparently the linter doesn't like the spaces around the `=` here:

```
$ make lint
flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
./sevsnpmeasure/id_block.py:162:11: E251 unexpected spaces around keyword / parameter equals
./sevsnpmeasure/id_block.py:162:13: E251 unexpected spaces around keyword / parameter equals
./sevsnpmeasure/id_block.py:163:11: E251 unexpected spaces around keyword / parameter equals
./sevsnpmeasure/id_block.py:163:13: E251 unexpected spaces around keyword / parameter equals
4     E251 unexpected spaces around keyword / parameter equals
4
make: *** [Makefile:7: lint] Error 1
```

can you please remove the spaces, run `make lint`, and then push the fix?

(I see now in the CI that it shows the warnings but still continues the build; I'll fix that separately.)

Reply:
Fixed